english/chickens/chickens.md

```diff
@@ -0,0 +1,22 @@
+You are on a farm. Middle of Iowa.
+You notice you are wearing overhauls. A worn down house stands behind you.
+
+You remember hearing someone call your name.
+But you dont know whether to trust yourself.
+As you turn to look at the house, you realize something is at your feet.
+You recognize it as your pet chicken...
+
+You look up and suddenly, hearing the voice again.
+The voice says "Come on, what is the deal!"
+
+You look down, and realize it is the chicken speaking...
+
+Do you:
+
+[Kick it as far and as fast as possible](https://www.youtube.com/watch?v=6VuwxijKof0)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.